.github/workflows/Release.yml

```diff
@@ -0,0 +1,25 @@
+name: Create Release
+
+on:
+  push:
+    tags:
+      - "v*" # Push events to matching v*, i.e. v1.0, v20.15.10
+
+jobs:
+  build:
+    name: Create Release
+    runs-on: ubuntu-latest
```

Comment:
@SyXuan should we consider locking the ubuntu version instead of "latest"? I am fine to follow the best practices in this space.

Reply:
done